environments/icds-cas/postgresql.yml

```diff
@@ -53,6 +57,10 @@ postgres_override:
     - contype: host
       users: TCLBKPADMIN
       netmask: 100.71.188.129/32
+    - contype: host
+      databases: replication
+      users: TCLBKPADMIN
+      netmask: 100.71.188.129/32
```

Comment:
This was missing when I rolled out the PG changes @rohit-dimagi is this required or just left over from some testing?